aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/ApplicationServiceImpl.java

```diff
@@ -278,6 +280,26 @@ public int publishToImaging(String applicationName, long sleepDuration, boolean
 
     }
 
+    @Override
+    public boolean checkServerFoldersExists(String pathToCheck) {
+        try {
+            FileCommandRequest fileCommandRequest = FileCommandRequest.builder().command("LS").path("SOURCES:" + Paths.get(pathToCheck)).build();
+            restApiService.postForEntity("/api/server-folders", fileCommandRequest, String.class);
+        } catch (ApiCallException e) {
+            return false;
+        }
+        return true;
+    }
+
+    @Override
+    public String downloadDeliveryReport(String appGuid, String versionGuid, String reportFilename) throws ApplicationServiceException {
+        try {
+            return restApiService.getForEntity("/api/applications/" + appGuid + "/versions/" + versionGuid + "/dmt-report/download", String.class);
+        } catch (ApiCallException e) {
+            throw new ApplicationServiceException("Failed to download the delivery report", e);
+        }
+    }
+
```

Comment:
No need to remove this, but I would prefer if you kept the scope of a PR to the specific ticket; this looks like a change from another ticket that should be separated. For this, nothing needs to be changed, just a reminder for other PRs :)

Reply:
got it
But was just part of a refactoring